assignment-1/Functions.java

```diff
@@ -0,0 +1,271 @@
+import java.io.*;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.toMap;
+
+public class Functions {
+
+    public static final String delimiter = ",";
+    public static ArrayList<Players> allPlayers = new ArrayList<Players>();
+    public static ArrayList<Team> allTeams = new ArrayList<Team>();
+    public static <Player> void readData(String csvFile) {
+        try {
+            File file = new File(csvFile);
+            FileReader fr = new FileReader(file);
+            BufferedReader br = new BufferedReader(fr);
+            String line = "";
+            String[] individualField;
+
+            //allPlayers
+
+            while ((line = br.readLine()) != null) {
```

Comment:
why would you need 2 loops?
you could have just extracted the header outside the while loop to ignore it